state_monitor:
  ros__parameters:
    sensors:
      - sensor1
      - sensor2
    check_interval: 5.0
